### ISSUE TITLE
How to train with a smaller net-input size such as (640,480)?

### ISSUE THREAD

**Issue by JackHenry1992** [roytseng-tw/Detectron.pytorch#26]
Thanks for sharing your great job.
By printing the size of image, I get 768x1344 of resnet50-fpn model.
However, in my case, I want to retrain this network using a smaller net-input size, such as 640x480.

I simply tried to modify the config file of `e2e_mask_rcnn_R-50-FPN_2x.yaml ` as follows: 
![image](https://user-images.githubusercontent.com/9390228/39192407-b2575efa-480b-11e8-8cf7-e112ee7a505f.png)

But during traing, it said `python double free or corruption` error:
![image](https://user-images.githubusercontent.com/9390228/39192512-e8895190-480b-11e8-9d6f-4de014df5229.png)

Can you give me some advices for solving this error? 
Have you trained with a smaller net-input size?

**Comment by roytseng-tw** [comment]
Hello, your error seems not related to this code. It should be fine to adjust `SCALES` and `MAX_SIZE` in config file.

**Comment by JackHenry1992** [comment]
Sorry to bother you，I still have this error  when I use smaller size for the training.
I also test the #3405283 commit and just modify the (scale, maxsize) to (480, 540), the same error
![image](https://user-images.githubusercontent.com/9390228/39464034-ce971294-4d4d-11e8-8ff7-3f59907e0f33.png)

Can you give some adivces?

**Comment by roytseng-tw** [comment]
I am able run the code by following command correctly `python tools/train_net_step.py --dataset coco2017 --cfg configs/e2e_mask_rcnn_R-50-FPN_1x.yaml --set TRAIN.SCALES "(480,)" TRAIN.MAX_SIZE 540`
I cannot reproduce your error.

**Comment by JackHenry1992** [comment]
Yeah, this error happend at serveral iterations (for me, it happend randomly)

**Comment by JackHenry1992** [comment]
I have re-ran this command 'python tools/train_net_step.py --dataset coco2017 --cfg configs/e2e_mask_rcnn_R-50-FPN_1x.yaml --set TRAIN.SCALES "(480,)" TRAIN.MAX_SIZE 540', and after 2981 steps, Segentation fault occured.
![image](https://user-images.githubusercontent.com/9390228/39503582-451de702-4df9-11e8-9507-5ea53006f3d2.png)

**Comment by roytseng-tw** [comment]
I recommend you to install a fresh environment of python and other needed packages. I doubt that something in your library may be corrupted.

**Comment by JackHenry1992** [comment]
I will try it later. But what is strange is that I have trained successfully with 800x1333. Is there difference between large size and small size? As I think, the kernel should be the same.

**Comment by roytseng-tw** [comment]
How's everything going ?

**Comment by JackHenry1992** [comment]
I'm not sure about this because this issue occured randomly, and I observe that `loss_rpn_bbox_fpn6=0` all the time, so I change `FPN.RPN_MAX_LEVEL=5`. Now I can run the whole training process occasionally without any crash. But it seems that this issue may happen at some uncertain steps. The training command which I use as follows:
![image](https://user-images.githubusercontent.com/9390228/39682293-a824dbe4-51e1-11e8-9f5a-3ed9c264d7ba.png)

**Comment by roytseng-tw** [comment]
Uh, your image size is relatively small. Not sure if there is any bug on boundary condition, but sorry I have no time for debugging this. Maybe you can try doing the same thing in Detectron and see if the same issue exists.
BTW, for fpn5 the feature map size is 270 / 16, and for fpn6 the size if half smaller.
